pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
   'typing-compat; python_version < "3.8"',
   "typing_extensions>=4.4.0",
   "uvicorn[standard]>=0.12,<1",
+  "datamodel-code-generator >=0.25"
```

Comment:
if we're going to use this we'd probably need to vendor it, it's not ideal to add new deps

Reply:
removed